crates/node/builder/src/launch/common.rs

```diff
@@ -115,8 +115,10 @@ impl LaunchContext {
     pub fn load_toml_config(&self, config: &NodeConfig) -> eyre::Result<reth_config::Config> {
         let config_path = config.config.clone().unwrap_or_else(|| self.data_dir.config());
 
-        let mut toml_config = confy::load_path::<reth_config::Config>(&config_path)
-            .wrap_err_with(|| format!("Could not load config file {config_path:?}"))?;
+        let mut toml_config: reth_config::Config = std::fs::read_to_string(&config_path)
+            .ok()
+            .and_then(|s| toml::from_str(&s).ok())
+            .unwrap_or_default();
```

Comment:
we need to propagate the error and not just ignore it to match the previous implementation

Reply:
Should be fixed, let me know. I think that for `std::fs::read_to_string(&config_path)` we should unwrap or default otherwise it will trigger a failure on `op` `p2p::can_sync` test due to `Could not read config file` if we do something like:

```rust
let config_content = std::fs::read_to_string(&config_path)
    .wrap_err_with(|| format!("Could not read config file: {:?}", config_path))?;

let mut toml_config: reth_config::Config = toml::from_str(&config_content)
    .wrap_err_with(|| format!("Could not parse config file: {:?}", config_path))?;
```